spict/R/report.R

```diff
@@ -40,8 +40,30 @@ latex.figure <- function(figfile, reportfile, caption=''){
 #' @return Nothing.
```

Comment:
This function returns `TRUE` or `invisible(NULL)` depending on `!keep.figurefiles`, not nothing! But maybe it should return something like the pdf filename with full path, so one could open it using `system(paste(getOption("pdfviewer"), outpdf), wait = FALSE)`. We can also have an argument to show the pdf file after creation.